Services/TermsOfService/classes/History/class.ilTermsOfServiceAcceptanceHistoryGUI.php

```diff
@@ -65,7 +65,8 @@ public function executeCommand() : void
         ) {
             $this->error->raiseError($this->lng->txt('permission_denied'), $this->error->MESSAGE);
         }
-
+    
+        // PHP8-Review: 'switch' with single 'case'
```

Comment:
I would like to keep it because this is some kind of `Code Convention` in `GUI` classes